ELSpace/Screens/Supporting Files/Translations/Base.lproj/Localizable.strings

```diff
@@ -0,0 +1,7 @@
+/* 
+  Created by Michał Czerniakowski on 05.06.2017.
+  Copyright © 2017 El Passion. All rights reserved.
+*/
+
+"google_login_button" = "Login With Google Account";
+"provide_access_token" = "Provide Your Access Token";
```

Comment:
At the planning session we agreed to log in only with google account. Remove direct token access flow.